### ISSUE TITLE
Satellite UI: Object lock modification on existing bucket

### ISSUE THREAD

**Issue by mobyvb** [storj/storj#6981]
Add a bucket-level action on the buckets page to allow enabling object lock on an existing, eligible bucket.

### AC:
* from the UI, object lock can be enabled on an existing bucket that has versioning enabled
* add a success notification once enabled

##### Bucket Actions: Enable Lock
<img width="290" alt="bucket-actions-enable-lock" src="https://github.com/storj/storj/assets/3217669/901851da-035b-4305-90dd-77697dc38749">

##### Dialog
<img width="515" alt="enable-bucket-lock" src="https://github.com/storj/storj/assets/3217669/80752148-7b5d-4180-99e5-cd0390990ab0">

**Comment by mobyvb** [comment]
cc @boshevski can you share the designs for this feature?

**Comment by boshevski** [comment]
Added designs and updated the AC

**Comment by NiaStorj** [comment]
As of 8/20/24: the comment above still stands

**Comment by ptdatta** [comment]
Hey @mobyvb @NiaStorj @boshevski is this open to work?

**Comment by NiaStorj** [comment]
9/10/24:
 Blocked because it's waiting on  PutObjectLockConfiguration S3 Action: https://github.com/storj/edge/issues/501

**Comment by NiaStorj** [comment]
10/28/24:

added back to the up next column since this should be deployed today

**Comment by mobyvb** [comment]
This ticket's AC are duplicated by https://github.com/storj/storj/issues/7171, which is in code review
we can close this ticket when that one is completed

**Comment by ferristocrat** [comment]
Closing as duplicate of #7171
